views/pages/admin/wishcards.pug

```diff
@@ -4,112 +4,15 @@ block title
     title See Wish Cards | Donate Gifts
 
 block content
```

Comment:
this whole block can be deleted, a component mounting point is already defined in the master.pug template for page components